src/LcdMenu.h

```diff
@@ -314,9 +328,9 @@ class LcdMenu {
         lcd->init();
         lcd->backlight();
 #else
-        uint8_t rs, uint8_t en, uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3,
+        uint8_t rs, uint8_t en, uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3, uint8_t backlight,
         MenuItem* menu) {
-        this->lcd = new LiquidCrystal(rs, en, d0, d1, d2, d3);
+        this->lcd = new LiquidCrystal(rs, en, d0, d1, d2, d3, backlight, POSITIVE);
```

Comment:
I have no statistic as to how many users still use the original LiquidCrystal lib by Arduino _(I still use it tho)_, if the number is minimal, I could divert to your proposal. Do you have any stats?

Reply:
my `platformio.ini` contains this entry in `lib_deps`: `fmalpartida/LiquidCrystal` 

Is that the 'original LiquidCrystal lib by Arduino'? I use what worked..

Reply:
This is the one by Arduino `arduino-libraries/LiquidCrystal@^1.0.7`